ultraLatency.py

```diff
@@ -198,6 +198,7 @@
         oh.setTriggerSource(ohboard,options.gtx,0x5) # GBT, 0x0 for GTX
         pass
 
+    oh.setTriggerThrottle(ohboard, options.gtx, options.throttle)
```

Comment:
I thought register value of 0 means no throttle.

Additionally why does it matter if we write to this register by default, from power on it should be set to not throttle, we're just writing this same value back into it no?

I don't see the problem.